src/sections/dataset/publish-dataset/PublishDatasetModal.tsx

```diff
@@ -83,14 +90,14 @@ export function PublishDatasetModal({
                 defaultChecked
                 onClick={handleVersionUpdateTypeChange}
                 name="update-type"
-                label={t('publish.minorVersion')}
+                label={t('publish.minorVersion') + ` (${nextMinorVersionString})`}
                 id="update-type-minor"
                 value={VersionUpdateType.MINOR}
               />
               <Form.Group.Radio
                 onClick={handleVersionUpdateTypeChange}
                 name="update-type"
-                label={t('publish.majorVersion')}
+                label={t('publish.majorVersion') + ` (${nextMajorVersionString})`}
                 id="update-type-major"
```

Comment:
in this both labels (minor and major) you can put everything inside template literals strings.
```
label={`${t('publish.minorVersion')} (${nextMinorVersionString})`}
```

Reply:
Some ideas also:
- change `<p>{t('publish.selectVersion')}</p>` to `<Form.Group.Text>{t('publish.selectVersion')}</Form.Group.Text>` so we keep form group texts helpers consistent styles.
- Remove margin: 10px; from .container class of PublishDatasetHelpText, try wrapping everything with a div or Stack component, to give some vertical spacing between PublishDatasetHelpText, License/Data use, Agreement and Update Version.